inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -311,9 +335,17 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
 
     writeBits(header, os);
 
+    for (auto &name : inputNames) {
+        writeBits(name.size(), os);
+        writeNBytes(name.c_str(), name.size(), os);
```

Comment:
I would 'waste' additional byte for null-terminating, as its not so size-critical.

Reply:
What's the reason to put null on end?

Reply:
e.g., It may be easier to read/debug this part of the exported file in generic text viewer sometime.
A bit of redundancy I know...